src/coreclr/tools/aot/crossgen2/Crossgen2RootCommand.cs

```diff
@@ -296,8 +296,8 @@ public static void PrintExtendedHelp(ParseResult _)
             Console.WriteLine(SR.DashDashHelp);
             Console.WriteLine();
 
-            string[] ValidArchitectures = new string[] {"arm", "armel", "arm64", "x86", "x64", "riscv64", "loongarch64"};
-            string[] ValidOS = new string[] {"windows", "linux", "osx", "ios", "iossimulator", "maccatalyst"};
+            string[] ValidArchitectures = ["arm", "armel", "arm64", "x86", "x64", "riscv64", "loongarch64", "wasm"];
+            string[] ValidOS = ["windows", "linux", "osx", "ios", "iossimulator", "maccatalyst", "wasi"];
```

Comment:
please add "browser" too, wasi is not top priority

Reply:
I'm swapping this to `browser` as that was really the intention for the target.